server_environment_autocreate/pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["whool"]
+build-backend = "whool.buildapi"
+
+#[project.optional-dependencies]
```

Comment:
While I like this idea, it should be removed to avoid future conflicts.
However, you could propose this improvement to the whool lib https://github.com/sbidoul/whool

Reply:
Opened sbidoul/whool#26

Reply:
Comment removed.